lib/br_danfe/danfe_lib/infadic_vol.rb

```diff
@@ -0,0 +1,103 @@
+module BrDanfe
+  module DanfeLib
+    class InfadicVol
+      def initialize(xml, pdf)
+        @xml = xml
+        @pdf = pdf
+        @y_position = 27.56
+      end
+
+      def render
+        render_volume_title
+        render_volume
+      end
+
+      private
+
+      def render_volume_title
+        @pdf.ibox 2.65, 12.45, 0.75, @y_position + 0.30, '', I18n.t(
+          'danfe.infAdic.vol.title'
+        ), size: 5, valign: :top, border: 0
+      end
+
+      def render_volume
+        volumes = 0
+        @y_position += 0.47
+
+        @xml.collect('xmlns', 'vol') do |det|
+          @det = det
```

Comment:
Isso não é um bom padrão, o correto é passar por parâmetro
```suggestion
          @det = det
```